BiweeklyMeetings/2022-07-05.md

```diff
@@ -0,0 +1,274 @@
+
+**M. Àngels** -:  Okay, cool. So I'll do that. I will add a comment after the call and mentioning the marketplace that had been approached.
+
+**Matthias** - :  Yeah. And there was also a mention in the newsletter, I think Robert pointed that out on the PR. On IOG newsletter, there was a mention of two CIPs if I recall correctly, there was this one.
+
+**M. Àngels** -:  Yes. He added the link in a comment.
+
+
+**Matthias** - :  Yeah, added the link, exactly. CIP 44 and CIP 54 were mentioned, that are touching CIP marketplaces or NFTs. 
+
+#### PR220
+[PR220 CCIP-0381? | Plutus support for pairings over curve Bls12-381](https://github.com/cardano-foundation/CIPs/pull/220)
+
+Yeah, okay. Next one, the pairing support over BLS 381, which we discussed a bit last week. And yeah, it was discussed with the Plutus' team, reviewed, adjusted many times. So yeah. So maybe the only thing is to see if the status shouldn't be changed to propose perhaps, I think it has reached a maturing of stage to be proposed now. So I will ask Nigo if he can do that. Yeah, and it has also just renamed the file to readme. So okay, beside a few minority editorial work, this one should be ready to go. So I will just comment directly. Okay. And I think... Okay, any comment on that one? No? Yes. I think we discussed it extensively last week already. 
+
+### Review 
+
+#### PR226
+[PR226 CIP-0044? | Additional factors for NFT market verification](https://github.com/cardano-foundation/CIPs/pull/226)
+
+**Matthias** - :Okay. So CIP 44 is next, and is also one we are waiting for NFT marketplaces feedback. Well, at least it is the main one.
+
+**Sebastien** -:  Yeah. I think the message asking for feedback was sent out a day or two, right? So I think we're still waiting.
+
+**Matthias** - :  So there's probably not much to do today on this one. Yeah. As we said last week, the ID looks good overall, but we had the question on whether or not this is useful for NFT marketplaces or endorsed by NFT marketplaces because it has to in order to become active. So we better include them early on in the feedback loop before merging or yeah, going with the proposed CIP. 
+
+
+#### PR242
+[PR242 CIP-0050? | Shelleys Voltaire decentralization update weekly update](https://github.com/cardano-foundation/CIPs/pull/242)
+
+**Matthias** - :So I guess we can just move to the next and last one on the agenda. Our favorite, CIP 50. I haven't got time to go through all the comments since last week, there has been a lot discussed. Again, there is a new member, I would say, in the discussions now, Robert, who is a mathematician working at Tesla, from what I got, who has got an interest in the CIP. So there has been quite a bunch of conversations between yeah, Rob and Michael, fortunately on the GitHub this time, on the GitHub issue. We also have created a physical channel for that one as we've been discussing last week, so a few more discussions have been going on here as well.
```

Comment:
```suggestion
**Matthias** - :So I guess we can just move to the next and last one on the agenda. Our favorite, CIP 50. I haven't got time to go through all the comments since last week, there has been a lot discussed. Again, there is a new member, I would say, in the discussions now, Robert, who is a mathematician working at Tesla, from what I got, who has got an interest in the CIP. So there has been quite a bunch of conversations between yeah, Rob and Michael, fortunately on GitHub this time, on the GitHub issue. We also have created a Discord channel for that one as we've been discussing last week, so a few more discussions have been going on here as well.
```